src/api/exporters/exportJson.js

```diff
@@ -0,0 +1,21 @@
+import ezs from 'ezs';
+import ezsBasics from 'ezs-basics';
+import ezsLocals from '../statements';
+
+ezs.use(ezsBasics);
+ezs.use(ezsLocals);
+
+const exporter = (config, fields, characteristics, stream) => {
+    return stream
+        .pipe(ezs('filterVersions'))
+        .pipe(ezs('filterContributions', { fields }))
+        .pipe(ezs('convertToJson', { fields }))
+        .pipe(ezs('jsonify'));
+};
+
+exporter.extension = 'json';
+exporter.mimeType = 'application/json';
+exporter.type = 'file';
+exporter.label = 'json';
```

Comment:
Maybe create a new translation key, because this one is translated to "JSON - with an array".